hr_attendance_calendar/models/hr_attendance_rules.py

```diff
@@ -12,11 +12,15 @@
 
 class HrAttendanceRules(models.Model):
     _name = 'hr.attendance.rules'
+    _sql_constraints = [('field_unique', 'unique(threshold)', 'This threshold '
+                                                              'already '
+                                                              'exist!')]
 
     ##########################################################################
     #                                 FIELDS                                 #
     ##########################################################################
-    name = fields.Char(string='Name')
-    threshold = fields.Float(string='Hours threshold in the day')
-    due_break = fields.Integer(string='Minimum break in minute')
-    due_break_total = fields.Integer(string='Total break due in minute')
+    threshold = fields.Float('Threshold (hour)',
```

Comment:
These are general rules ? I can't figure out how it will be used. Will you have only one record with the rules or can there be multiple rules ? How does it work?